biscuit-capi/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "biscuit-capi"
-version = "6.0.0-beta.3"                                    # Should keep the same version as biscuit-auth
+version = "6.0.0-beta.3"                                     # Should keep the same version as biscuit-auth
```

Comment:
regarding the version sync, does it still make sense, or could we decouple the two?